spec/controllers/rails_admin/main_controller_spec.rb

```diff
@@ -73,14 +73,14 @@ def get(action, params)
     context 'using mongoid, not supporting joins', mongoid: true do
       it 'gives back the remote table with label name' do
         controller.params = {sort: 'team', model_name: 'players'}
-        expect(controller.send(:get_sort_hash, RailsAdmin.config(Player))).to eq(sort: 'players.team_id', sort_reverse: true)
+        expect(controller.send(:get_sort_hash, RailsAdmin.config(Player))).to match(sort: /.?players.?\..?team_id.?/, sort_reverse: true)
       end
     end
 
     context 'using active_record, supporting joins', active_record: true do
       it 'gives back the local column' do
         controller.params = {sort: 'team', model_name: 'players'}
-        expect(controller.send(:get_sort_hash, RailsAdmin.config(Player))).to eq(sort: 'teams.name', sort_reverse: true)
+        expect(controller.send(:get_sort_hash, RailsAdmin.config(Player))).to match(sort: /.?teams.?\..?name.?/, sort_reverse: true)
```

Comment:
Maybe I should also add a regression spec with a model with capitalized fields? Where would I put it?

Reply:
Somewhere around here will be good, as this fix is about `#get_sort_hash`.
Also you can create RDBMS-specific test cases like:
https://github.com/railsadminteam/rails_admin/blob/0e12e5b465997c14d5b7a4d500a0d4cebed21aa9/spec/rails_admin/support/csv_converter_spec.rb#L69-L75